fa20/09-10-meeting00/meeting00.md

```diff
@@ -1,2 +1,6 @@
-<!-- TODO Add Meeting Notes/Contents here -->
-<!-- NOTE Refer the Documentation if you're unsure how to format/add to this. -->
+# Project Brainstorm
```

Comment:
Forgot to state this – there should be no H1's in the `*.md` files. The website already has an H1 ready (it's the meeting title).